TurkishFilter/sections/specific.txt

```diff
@@ -4613,6 +4613,7 @@ copeland.club##div[class^="copel-icerikten-"] > a[target="_blank"] > img
 ||vatandanhaber.com//files/uploads/walkertone.png
 ||vidout.net/vidoutMedia/ads/
 ||www.bolununsesi.com/ilangorsel/
+||analiz.r10.net/asset/images/banners/
```

Comment:
Domain can be added to the end of existing `/images/banners/*$domain=haber2000.com|enerjivetesisat.com`